freeipa/tests/delete_user.json

```diff
@@ -2,10 +2,6 @@
     "body": {
         "action": "delete_user",
         "connection": {
-            "credentials": {
```

Comment:
Add the credential field back and replace the nulls with empty strings. Do the same for server and all other tests.